src/secret.ts

```diff
@@ -154,6 +164,10 @@ export class Secret extends base.Resource implements ISecret {
   public getStringData(key: string): string | undefined {
     return this.stringData[key];
   }
+
+  envValue(key: string, options?: EnvValueFromSecretOptions): EnvValue {
```

Comment:
Just for consistency:

```suggestion
public  envValue(key: string, options?: EnvValueFromSecretOptions): EnvValue {
```